package-lock.json

```diff
@@ -0,0 +1,1779 @@
+{
```

Comment:
I'm not sure why this file wasn't previously committed, but I figured it would be fine to include in this PR. 

From the [npm docs](https://docs.npmjs.com/configuring-npm/package-lock-json.html):
> This file is intended to be committed into source repositories

Reply:
That's fine. Just an oversight.